src/common/platform/unicode.hpp

```diff
@@ -8,7 +8,6 @@ struct UNICODE_STRING
 {
     USHORT Length;
     USHORT MaximumLength;
-    uint32_t _Padding;
```

Comment:
i think this padding was needed for platforms with a different data layout. i might be wrong.